sample-apps/spark/src/main/java/com/amazon/sampleapp/App.java

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Remove license headers from sample apps and add a LICENSE file in the sample-apps folder saying these are MIT-0 licensed because they are sample apps

Reply:
I guess we'll need to tweak the spotless setting in the `sample-apps` project to fix the license. Can be followup though